v6/v6_interface.ml

```diff
@@ -20,17 +20,22 @@ let uri () = "file:" ^ !default_path
 
 type debug_info = string
 
+type edition_info = {
```

Comment:
It does contain a couple of different things, but all related to the edition that was applied. I'm fine with `edition_info`.